### PR TITLE
Add paint debug window

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Should be added to the debug drop down

File: src/windows/top_toolbar.c [hunk 1]
```diff
@@ -2988,6 +2999,13 @@ void top_toolbar_debug_menu_dropdown(short dropdownIndex)
 			break;
 		case DDIDX_SCENARIO_OPTIONS:
 			window_editor_scenario_options_open();
+				break;
+			case DDIDX_DEBUG_PAINT:
```
Comment (duncanspumpkin) [inline src/windows/top_toolbar.c]: Indentation is wrong for this section.

File: src/windows/debug_paint.c [hunk 2]
```diff
@@ -34,8 +34,8 @@ enum WINDOW_DEBUG_PAINT_WIDGET_IDX
 	WIDX_TOGGLE_SHOW_BOUND_BOXES,
 };
 
-static const int WINDOW_WIDTH = 200;
-static const int WINDOW_HEIGHT = 8 + 15 + 15 + 11 + 8;
+#define WINDOW_WIDTH	200
+#define WINDOW_HEIGHT	8 + 15 + 15 + 11 + 8
```
Comment (IntelOrca) [inline src/windows/debug_paint.c]: Very advised that you surround expressions of `#define` in parentheses.

**Comment by marijnvdwerf** [comment]
@IntelOrca Satisfied?

**Comment by danidoedel** [comment]
@marijnvdwerf Isn't `STR_SHORTCUT_DEBUG_PAINT_TOGGLE` missing here: https://github.com/OpenRCT2/OpenRCT2/blob/develop/src/windows/shortcut_keys.c#L136? Ignore me if I'm wrong ;)

**Comment by marijnvdwerf** [comment]
Oops. My bad!

**Pull request merged**